mruby_test.go

```diff
@@ -19,6 +19,16 @@ func TestMrbArena(t *testing.T) {
 	mrb.ArenaRestore(idx)
 }
 
+func TestMrbModule(t *testing.T) {
+	mrb := NewMrb()
+	defer mrb.Close()
+
+	module := mrb.Module("Kernel")
+	if module == nil {
+		t.Fatal("module was nil and should not be")
+	}
```

Comment:
Can you also add a negative test? (when a module that doesn't exist is requested)